### PR TITLE
Added shape_pixel property

### PR DESCRIPTION
**Pull request by astrofrog**
This is the proposed ``shape_pixel`` property

### CODE REVIEW

**Comment by Cadair** [comment]
lgtm

**Comment by Cadair** [comment]
is the fact it's C-ordered implicit because it's somewhere else in the APE? or should we mention it?

**Comment by astrofrog** [comment]
Are we saying anything about C order? I think the only requirement is that the shape is given in the same order than the pixel coordinates should be given when converting to world?

**Comment by Cadair** [comment]
Ah I was getting confused with 0-indexing. 👍

File: APE14.rst [hunk 1]
```diff
@@ -144,6 +144,20 @@ low-level API recommends:
             The number of axes in the world coordinate system
             """
 
+        @property
+        def shape_pixel(self):
+            """
+            The shape of the data that the WCS applies to.
```
Comment (eteq) [inline APE14.rst]: Maybe add ``(optional)`` or something at the end of this line?  I suspect many will read the lead sentence but not the paragraph...

Also, although it's pretty strongly implied by being a ``shape*``, perhaps say something like "a tuple of length n_wcs"?

**Review by eteq — APPROVED** [review]
Minor wording suggestion, but otherwise cool with it.

**Comment by keflavich** [comment]
lgtm.  I like @eteq's suggestions.

**Comment by parejkoj** [comment]
I like it, with @eteq suggestions. @r-owen  do you have any thoughts?

File: APE14.rst (hunk 1 above)
Comment (astrofrog) [reply APE14.rst]: Done!

**Comment by astrofrog** [comment]
I'll go ahead and merge this, and @r-owen, feel free to comment on this on the main APE PR if you have any thoughts - thanks!

**Pull request merged**